analysis/wikitrust_algorithms/db/models.py

```diff
@@ -0,0 +1,22 @@
+from mongoengine import *
```

Comment:
We need to discuss this.  I had begun writing the models in PyDal, and we need to discuss the idea of switching to mongoengine.  Personally I prefer to stick with PyDal, which enables us to use a wide variety of backend storage infrastructures.

Reply:
Luke has been doing the databases, so he's best suited to answer this one. @LukeJHarvey